AppCenterCrashes/AppCenterCrashes/Internals/Util/MSApplicationForwarder.m

```diff
@@ -24,7 +24,7 @@ static BOOL isApplicationForwarderEnabled() {
 static NSString *const kMSCrashOnExceptionsKey = @"NSApplicationCrashOnExceptions";
 
 static BOOL isCrashOnExceptionsEnabled() {
-  NSNumber *crashOnExceptions = [MS_USER_DEFAULTS objectForKey:kMSCrashOnExceptionsKey];
+  NSNumber *crashOnExceptions = [[NSUserDefaults standardUserDefaults] objectForKey:kMSCrashOnExceptionsKey];
```

Comment:
I would add a comment to explain that it's a system key so we don't use the App Center wrapper.